setup.py

```diff
@@ -20,12 +20,11 @@
               "jsmol-bokeh-extension~=0.2.1",
               "pandas~=0.24.2",
               "requests~=2.21.0",
-              "panel~=0.6.4",
-              "param~=1.9.2",
+              "panel~=0.8",
+              "param~=1.9.3",
               "jupyter",
               "graphviz~=0.13",
-          ],
-          extras_require={
```

Comment:
Can we put this back?

This is for developers only, no need to install this on materials cloud